nft_ingester/src/api/builder.rs

```diff
@@ -98,6 +99,21 @@ impl RpcApiBuilder {
         });
         module.add_alias("getAssetsByCreator", "get_assets_by_creator");
 
+        let cloned_api = api.clone();
+        module.add_method("get_nft_editions", move |rpc_params: Params| {
+            let api = cloned_api.clone();
```

Comment:
we already did a clone for this variable, seems like a redundant clone. for some reason rust-analyze is not working, so I can confirm this. (all the cases in that impl use the same pattern, which may be redundant)

Reply:
The two clones are necessary because the closure passed to module.add_method needs to own the api instance. The first api.clone() creates a new Arc instance that is then moved into the closure.

Reply:
but